pkgs/applications/graphics/gthumb/default.nix

```diff
@@ -36,11 +35,12 @@ in stdenv.mkDerivation rec {
   passthru = {
     updateScript = gnome3.updateScript {
       packageName = pname;
+      versionPolicy = "none";
```

Comment:
This seems to follow GNOME versioning policy:

https://mail.gnome.org/archives/gthumb-list/2019-February/msg00000.html

Reply:
wow, good catch! I had only looked here: https://gitlab.gnome.org/GNOME/gthumb/blob/master/NEWS

Reply:
Yeah, they often assume everyone is familiar with the versioning schema and then there are exceptions :confused: